src/angular-public/textexpand/textexpand-collapsed/textexpand-collapsed.component.ts

```diff
@@ -1,20 +1,13 @@
-import { ChangeDetectionStrategy, Component, HostBinding } from '@angular/core';
+import { Directive } from '@angular/core';
 
-@Component({
+@Directive({
   selector: 'sbb-textexpand-collapsed',
-  templateUrl: './textexpand-collapsed.component.html',
-  styleUrls: ['./textexpand-collapsed.component.css'],
-  changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {
+    class: 'sbb-textexpand-collapsed',
+    '[attr.hidden]': '_hidden',
```

Comment:
```suggestion
    '[attr.hidden]': '_hidden ? true : null',
```

css styling won't work otherwise